chapter-09/recipe-03/cxx-example/CMakeLists.txt

```diff
@@ -45,11 +52,6 @@ add_test(
   NAME
     python_test
   COMMAND
+    ${CMAKE_COMMAND} -E env ACCOUNT_MODULE_PATH=$<TARGET_FILE_DIR:account>
```

Comment:
Great find :+1: I've read somewhere that fiddling with `PYTHONPATH` can make starting the interpreter slower, so it's definitely a last resort trick to pull.